Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name          = "test-case"
 version       = "3.0.0"
-edition       = "2018"
+edition       = "2021"
```

Comment:
We should also bump edition in `crates/test-case-macros` and `crates/test-case-core`.

Reply:
Done in all crates in the repository.